### DISCUSSION TITLE
file_info icon

### DISCUSSION THREAD

**Discussion by 3n3l** [famiu/feline.nvim#300] [Ideas]
Hey,
you can disable the icon for the file_type provider and change it for the git_branch provider,  is there some way to change/disable the icon for the file_info provider which I am not seeing?

If not I would make a PR to change/disable the icon via the opts table for the file_info provider.

**Comment by famiu** [comment]
Just set the `icon` value of the component to an empty string?

**Reply by 3n3l** [reply]
Does not work for the file_info for me, I set it in the component the same way I set it for git_branch, where it works just fine.

Even if it would work that way, why is it so different than the file_type provider?
It would be a lot more intuitive to just have the same behavior for every provider, at least in the same category.

**Reply by famiu** [reply]
Can you show me your config?
